src/hooks/core/chat/useDMMessage.ts

```diff
@@ -80,12 +74,13 @@ const useDMMessage = () => {
   const sendMessageDM = async (
     id,
     source: 'post' | 'comment',
-    channelCategory: ChannelCategory
+    channelCategory: ChannelCategory,
+    feedId: string
   ) => {
     const initChat =
       channelCategory === 'ANONYMOUS'
-        ? await initChatFromPostAnon({source, id})
-        : await initChatFromPost({source, id});
+        ? await initChatFromPostAnon({source, id, feedId})
+        : await initChatFromPost({source, id, postId: feedId});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendMessageDM` function has been updated to include a new parameter `feedId`. This is used in the calls to `initChatFromPostAnon` and `initChatFromPost`. However, it's not clear from this hunk whether all calls to `sendMessageDM` throughout the codebase have been updated to include this new argument. If not, this could lead to runtime errors. Please ensure that all invocations of `sendMessageDM` are updated to pass the `feedId` argument.
```diff
- const sendMessageDM = async (
-   id,
-   source: 'post' | 'comment',
-   channelCategory: ChannelCategory
- ) => {
+ const sendMessageDM = async (
+   id,
+   source: 'post' | 'comment',
+   channelCategory: ChannelCategory,
+   feedId: string
+ ) => {
    const initChat =
      channelCategory === 'ANONYMOUS'
       ? await initChatFromPostAnon({source, id, feedId})
       : await initChatFromPost({source, id, postId: feedId});
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->